app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -11,15 +11,28 @@ def index
   end
 
   def create
-    @plan_app = PlanningApplication.create(full_planning_application_params)
-    if @plan_app.save!
-      @plan_app.create_policy_evaluation
-      render json: { "id": "#{@plan_app.reference}",
-                    "message": "Application created" }, status: 200
+    site_id = create_site
```

Comment:
🏄  `create_site` isn't quite an accurate method name here, as it's a find and create. In fact, I'd say that it doesn't need to be it's own method, as the code within the method is more descriptive than any method name I can think of.